dpnp/dparray.pyx

```diff
@@ -33,23 +33,26 @@ using USB interface for an Intel GPU device.
 
 """
 
+import warnings
 
 from libcpp cimport bool as cpp_bool
 
 from dpnp.dpnp_iface_types import *
-from dpnp.dpnp_iface import *
```

Comment:
Is importing of this module fixed by changes on lines 52-55 without removal of `from dpnp.dpnp_iface import *`.

Reply:
No, it didn't help, the same exception raised.